lib/taiko.js

```diff
@@ -1937,13 +1937,19 @@ function dropDown(attrValuePairs, ...args) {
         exists: exists(getIfExists(get), descEvent),
         description: desc(selector, 'label', 'Dropdown'),
         select: async (value) => {
-
             const nodeId = (await getIfExists(get)())[0];
             if (!nodeId) throw `${desc(selector, 'label', 'DropDown')} not found`;
             if (headful) await highlightElemOnAction(nodeId);
 
             function selectBox(value) {
                 let found_value = false;
+                if(this.options.length > value.index){
```

Comment:
Why value.index and not just value which would simplify the API

From:
```
await dropDown(below('Reason')).select({index:1});
```

To:
```
await dropDown(below('Reason')).select(1);
```

Reply:
@saikrishna321 Then there is a possibility that it may collide with other element's value... Meaning consider two elements, one element with index 1 and the other with value 1. Which one do you select then?